### DISCUSSION TITLE
Custom headers with Netlify

### DISCUSSION THREAD

**Discussion by darrenjansson** [sveltejs/kit#3036] [Q&A]
If I do a `curl` on my HTTP Netlify url I get the custom headers on the 301 redirect but not then 200 response it goes to, and if I load the HTTPS directly I don't get the custom headers at all.

Am I right in thinking that [adapter-netlify](https://github.com/sveltejs/kit/tree/master/packages/adapter-netlify) doesn't currently handle any custom headers set via a `_headers` file, and any headers set in `netlify.toml` wouldn't work either?

**Comment by swgn** [comment]
I have some custom headers working just fine on my netlify site. I'm setting via the following lines at the end of my `netlify.toml` file (I'm not using a stand-alone `_headers` file to add these):

```
[[headers]]
  for = "/*"
  [headers.values]
  Strict-Transport-Security = "max-age=63072000; includeSubDomains; preload"
  X-Frame-Options = "DENY"
  X-XSS-Protection = "1; mode=block"
  X-Content-Type-Options = "nosniff"
  Referrer-Policy = "strict-origin-when-cross-origin"
  Permissions-Policy = "camera=(), geolocation=(), microphone=()"
```
